event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateWithSyntheticDataTest.java

```diff
@@ -45,13 +43,12 @@ void setup() {
   // The parser should be able to handle documents with 10 000 contributors in 30 seconds.
   // This test case is a bit more generous because the GitHub Actions test runner is underpowered.
   @ParameterizedTest
-  @Timeout(value = 30, unit = TimeUnit.SECONDS)
+  @Timeout(value = 45, unit = TimeUnit.SECONDS)
```

Comment:
This is an unrelated change, but the GitHub actions runner is so slow that this test was already close to the limit and has now started breaking it. We should revisit this at some point...